scripts/start

```diff
@@ -29,6 +29,8 @@ check_dependencies fswatch
 check_dependencies rsync jq curl
 
 UMBREL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))/.."
+UMBREL_LOGS="${UMBREL_ROOT}/logs"
+mkdir -p "${UMBREL_LOGS}"
```

Comment:
Do you think this is required since we're explicitly committing the logs directory?

Reply:
I think it's good to have, ideally we should always ensure folders exist before writing to them.

Otherwise if someone does `rm -rf logs/` the Umbrel service will fail to start.

Reply:
I agree, but isn't that the same for every other directory? Like if someone did `rm -rf electrs/` or `rm -rf bitcoin/`, it would fail to start too (and/or result in errors)?

Reply:
Yeah I suppose. I added this before checking `logs/` into git so I guess it's fine to remove it.